packages/cli/commands/sandbox/create.js

```diff
@@ -151,6 +151,8 @@ exports.handler = async options => {
   } catch (err) {
     debugErrorAndContext(err);
 
+    trackCommandUsage('sandbox-create', { success: false }, accountId);
```

Comment:
To get to an accurate number of successful creates, we'll just subtract the `success: false` events from the grand total. Same for deletes